src/Routes/Router.php

```diff
@@ -0,0 +1,272 @@
+<?php
+/**
+ * Class Router
+ *
+ * Class that is responsible for registering routes.
+ *
+ * @package Tribe\Project\Routes
+ */
+
+declare( strict_types=1 );
+
+namespace Tribe\Libs\Routes;
+
+use Tribe\Libs\Container\Abstract_Subscriber;
+use Tribe\Libs\Routes\Abstract_Route;
+
+/**
+ * Class to register routers for normal and REST API endpoints.
+ */
+class Router extends Abstract_Subscriber {
```

Comment:
Ah, I see now that you're extending the Container's subscriber for this. I'm not sure this is a very good idea, in the sense that responsibility for a subscriber should be to register instances to WordPress hooks. This should really be its own stand alone/testable class that is instead registered for the proper hooks in the Router_Subscriber.